src/status_im/contexts/communities/actions/accounts_selection/style.cljs

```diff
@@ -16,13 +16,9 @@
    :padding-bottom     4
    :padding-horizontal screen-horizontal-padding})
 
-(defn bottom-actions
-  []
-  {:position           :absolute
-   :background-color   (colors/theme-colors colors/white colors/neutral-95)
-   :bottom             0
-   :left               0
-   :right              0
-   :padding-horizontal screen-horizontal-padding
-   :padding-vertical   12
-   :flex               1})
+(def bottom-actions
+  {:flex-direction    :column
+   :align-items       :flex-end
+   :margin-horizontal 15
```

Comment:
We need to add `:padding-top 12`

Missing white background above the button:
<img src="https://github.com/status-im/status-mobile/assets/19279756/b38a8465-e1e6-476b-a1f1-68696e659bac" width="300"/>